src/box/lua/config/applier/credentials.lua

```diff
@@ -559,16 +559,20 @@ local function sync_privileges(credentials, obj_to_sync)
                     obj_to_sync.type, obj_to_sync.name)
     end
 
-    -- Drop missed privilege alerts.
+    -- Prepare to drop missed privilege alerts.
```

Comment:
Nit:

> This **could** happen if there are **missed_** credential warnings

Reply:
Fixed.

Reply:
> This **could** happen

Reply:
I rewrote this commit message.

Reply:
Thank you! Now the problem that is solved is clear.